card/lib/card/cache/temporary.rb

```diff
@@ -0,0 +1,38 @@
+class Card::Cache::Temporary
+  attr_reader :store
+
+  def initialize
+    @store = Hash.new
+  end
+
+  def read key
+    return unless @store.has_key? key
+    @store[key]
+  end
+
+  def write key, value
+    @store[key] = value
+  end
+
+  def fetch key, &block
+    read(key) || write(key, block.call)
+  end
+
+  def delete key
+    @store.delete key
+  end
+
+  def dump
+    @store.each do |k, v|
+      p "#{k} --> #{v.inspect[0..30]}"
+    end
+  end
+
+  def reset hard=false
```

Comment:
Unused method argument - `hard`. If it's necessary, use `_` or `_hard` as an argument name to indicate that it won't be used. You can also write as `reset(*)` if you want the method to accept any arguments but don't care about them.